### DISCUSSION TITLE
More tools!

### DISCUSSION THREAD

**Discussion by Creators-722** [ScratchAddons/ScratchAddons#2208] [Ideas]
# More tools
hey, I'm a user of scratchaddons! I wanted to suggest some features like:-

- [ ] An extra but special block category
**It may have its very own working function**
- [ ] More costume editor tools
**Those tools are boring!**
- [ ] Another theme?
**Just LIGHT and DARK themes are fine, pleeeeasseeeee add another one*?*

**Comment by mxmou** [comment]
Additional blocks will not be added because they wouldn't work for other users.

More customizability will be added in version 1.13, see #1805.

**Comment by cobaltt7** [comment]
1. rejected
2. like what?
3. #1805
